compiler/modules/types/list.bal

```diff
@@ -64,10 +64,10 @@ public class ListDefinition {
     *Definition;
     private RecAtom? rec = ();
 
-    private ComplexSemType? semType = ();
+    public SemType? semType = ();
```

Comment:
Why change to `public`?

Reply:
Mistake. Changed for debug logging. Fixed.